describeconfigs.go

```diff
@@ -0,0 +1,157 @@
+package kafka
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/segmentio/kafka-go/protocol/describeconfigs"
+)
+
+// DescribeConfigsRequest represents a request sent to a kafka broker to describe configs
+type DescribeConfigsRequest struct {
+	// Address of the kafka broker to send the request to.
+	Addr net.Addr
+
+	// List of resources to update
+	Resources []DescribeConfigRequestResource
+
+	// required API version is no less than v1
+	IncludeSynonyms bool
+
+	// required API version is no less than v3
+	IncludeDocumentation bool
+}
+
+type DescribeConfigRequestResource struct {
+	// Resource Type
+	ResourceType int8
+
+	// Resource Name
+	ResourceName string
+
+	// Configs is a List of configurations that need to update
+	ConfigNames []string
+}
+
+// DescribeConfigsResponse represents a response from a kafka broker to a describe config request.
+type DescribeConfigsResponse struct {
+	// The amount of time that the broker throttled the request.
+	//
+	// This field will be zero if the kafka broker did no support the
+	// DescribeConfigs API in version 2 or above.
+	Throttle time.Duration
+
+	// Resources
+	Resources []DescribeConfigResponseResource
+}
+
+// DescribeConfigResponseResource
+type DescribeConfigResponseResource struct {
+	// Resource Type
+	ResourceType int8
+
+	// Resource Name
+	ResourceName string
+
+	// ErrorCode
+	ErrorCode int16
+
+	// ErrorMessage
+	ErrorMessage string
+
+	// ConfigEntries
+	ConfigEntries []DescribeConfigResponseConfigEntry
+}
+
+// DescribeConfigResponseConfigEntry
+type DescribeConfigResponseConfigEntry struct {
+	ConfigName          string
+	ConfigValue         string
+	ReadOnly            bool
+	IsDefault           bool
```

Comment:
document `isDefault` is only v0

Reply:
So here how can i deal with it?

Reply:
I was thinking something similar to `DescribeConfigsRequest.IncludeSynonyms` have a comment, something like

```
// Ignored if API version is greater than v0
```